src/Bus.js

```diff
@@ -105,11 +105,11 @@ class Bus {
             this._mozaik.logger.info(`Setting timer for "${ requestId }"`);
             this.subscriptions[requestId].timer = setInterval(() => {
                 this.processApiCall(requestId, callFn, request.params);
-            }, 10000);
+            }, 100000);
```

Comment:
Yes, it does, it helps for local testing, I added the **WIP** flag to this PR, won't be part of it at the end